src/components/NavigationBar.vue

```diff
@@ -20,11 +20,12 @@
                         <router-link class="nav-link text-white" :to="{name: 'home'}">Info</router-link>
                     </li>
                 </ul>
+                <hr class="d-xl-none">
                 <ul class="navbar-nav mb-2 mb-lg-0">
                     <li class="nav-item">
-                        <router-link class="nav-link text-white" :to="{name: 'home'}">Inloggen</router-link>
+                        <router-link class="nav-link text-white" :to="{name: 'login'}">Inloggen</router-link>
                     </li>
-                    <router-link class="btn bg-white" to="{name: 'home'}">Registreer</router-link>
+                    <router-link class="btn bg-white" :to="{name: 'register'}">Registreer</router-link>
```

Comment:
Do we not need an `li` element around this?

Reply:
I more referencing to the fact that all the `router-link` elements were wrapped in a `li` element because they are located in a `ul` and the link for the register page was the only one not wrapped in a `li` element. Thus my question

Reply:
Ahh. It's not needed but a ul with out a li elemet is weird i will add it for both